examples/XX-anomaly-detection.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<p>by state 1 or 2 I imagine you mean value 1 or value 2?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/1518/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
Can't i keep state?

Reply:
<p>You could but this is slightly confusing. In general, state != value. The state can be used to enumerate values, for instance you could say:</p><p> </p><ul><li>state 1 corresponds to value 0</li><li>state 2 corresponds to value 1</li><li>state 3 corresponds to value 2</li><li>state 4 corresponds to value 3</li></ul><p> </p><p>Here you seem to be using the words state and value interchangeably. It's fine because this case is very simple, but I think it would be a bit cleaner to use the right terms. </p><p> </p><p>You did it well when you defined the initial synthetic data as follows above:</p><p>"</p><ul><li>State 1: component1=0, component2=1</li><li>State 2: component1=1, component2=0</li></ul><p>"</p>